### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by HaozhouPang** [wangqiang9/Finetune-SVD#6]

**Comment by HaozhouPang** [comment]
Hello, Thanks for sharing the training script, I have a couple of questions regarding the training details.

**Comment by HaozhouPang** [comment]
Also, according to the report by SVD, the SVD-MV model was conditioned on the elevation angle, but I didn't see that in your current implementation, could you please clarify that or correct me if I'm wrong. 

Much appreciated.

**Comment by wangqiang9** [comment]
@HaozhouPang Hi, As described in readme, I used 1k videos for training. Additionally, if you can adjust your perspective through the generated dataset, it can be useful.

**Comment by HaozhouPang** [comment]
> @HaozhouPang Hi, As described in readme, I used 1k videos for training. Additionally, if you can adjust your perspective through the generated dataset, it can be useful.

Thanks for your reply!

I understand the current comparison result in read was achieved by finetuning on a small (1k) dataset, but how many steps / gpu hours did you train? What's the loss magnitude? 

For adjusting the perspective, did you mean to adjust the elevation/azumith  when rendering the video?
